src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelEvents.cs

```diff
@@ -56,17 +56,7 @@ public virtual void OnRequestSave3DImage(object sender, ImageSaveEventArgs e)
             {
                 RequestSave3DImage(this, e);
             }
-        }
-
-        public event EventHandler RequestScaleFactorDialog;
-        public virtual void OnRequestScaleFactorDialog(object sender, EventArgs e)
```

Comment:
Since this is partial class of `DynamoViewModel`. Let's obsolete instead of fully removal

Reply:
@QilongTang I re-added the events and obsolete them in the commit: https://github.com/DynamoDS/Dynamo/pull/11657/commits/10e7dfb9cc42b4bf664997abb54706a1f3004c98